addOns/pscanrules/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
     - Application Error Disclosure
     - Information Disclosure - Suspicious Comments
     - Username Hash Found
+- Updated Timestamp Disclosure Scan Rule to skip Javascript files when Alert Threshold is set to High
```

Comment:
"Javascript " ➡️ "JavaScript"
and add a period at the end.

The help content: https://github.com/zaproxy/zap-extensions/blob/124016cff14eb93673c390353251103fd2bbd2c9/addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html#L477-L481 should also be updated. (There are other threshold notes in the file.)